src/geojson/geojson.ml

```diff
@@ -31,6 +31,14 @@ module Make (J : Intf.Json) = struct
   module Geometry = struct
     type json = J.t
 
+    let keys_in_use = [ "type"; "coordinates" ]
```

Comment:
All GeoJSON objects should also remove "bbox" I think -- that's where the test failure is coming from. (https://datatracker.ietf.org/doc/html/rfc7946#section-5)

Reply:
yeah got it , me aslo too silly :))